<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResidualViewConfigWidget</class>
 <widget class="QWidget" name="ResidualViewConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Residual View Parameters</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="2" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="interpolationModeCheckBox">
        <property name="text">
         <string>Interpolate Model on Observation structure</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Observation</string>
        </property>
        <property name="buddy">
         <cstring>observationCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="observationCombo"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>&amp;Model</string>
        </property>
        <property name="buddy">
         <cstring>modelCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="modelCombo"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Line of Sight</string>
        </property>
        <property name="buddy">
         <cstring>losX</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="DoubleSpinBox" name="losX">
           <property name="toolTip">
            <string>East</string>
           </property>
           <property name="minimum">
            <double>-1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DoubleSpinBox" name="losY">
           <property name="toolTip">
            <string>North</string>
           </property>
           <property name="minimum">
            <double>-1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DoubleSpinBox" name="losZ">
           <property name="toolTip">
            <string>Vertical</string>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
           <property name="minimum">
            <double>-1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>O&amp;bservation Scale</string>
        </property>
        <property name="buddy">
         <cstring>observationScale</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QSpinBox" name="observationScale">
        <property name="prefix">
         <string>· 10^</string>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Model &amp;Scale</string>
        </property>
        <property name="buddy">
         <cstring>modelScale</cstring>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QSpinBox" name="modelScale">
        <property name="prefix">
         <string>· 10^</string>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item row="14" column="0" colspan="2">
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string>&amp;Update</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/DoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>interpolationModeCheckBox</tabstop>
  <tabstop>observationCombo</tabstop>
  <tabstop>modelCombo</tabstop>
  <tabstop>losX</tabstop>
  <tabstop>losY</tabstop>
  <tabstop>losZ</tabstop>
  <tabstop>observationScale</tabstop>
  <tabstop>modelScale</tabstop>
  <tabstop>updateButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
